### DISCUSSION TITLE
Contact points detection

### DISCUSSION THREAD

**Discussion by camilla-agabiti** [SofaDefrost/SoftRobots.Inverse#3] [Q&A]
Hi :)

I am working on whole-arm grasping, and I am using the trunk-like soft arm and a cube as the object to be grasped. My question is: which is the easiest way in SOFA to detect the positions of contact points (x,y,z coordinates) at a certain time?

Thank you so much!
Camilla

**Comment by EulalieCoevoet** [comment]
Hello Camilla, 

I think your question is not really related to the plugin, right ? It's more a SOFA question, so would you mind asking it on this channel instead https://github.com/sofa-framework/sofa/discussions ? Maybe your answer is already there. 

Eulalie
